### DISCUSSION TITLE
Think like a programmer

### DISCUSSION THREAD

**Discussion by SouzaHenrique** [pilhacheia/clube-do-livro-dev#9] [Books]
Pense como um programador: uma introdução à solução criativa de problemas (edição em inglês) 1ª Edição, eBook Kindle. Encontre todos os livros, leia sobre o autor, e muito mais. O verdadeiro desafio da programação não é aprender a sintaxe de uma linguagem - é aprender a resolver problemas de forma criativa para que você possa construir algo excelente.

Buy : https://www.amazon.com.br/Think-Like-Programmer-Introduction-Creative-ebook/dp/B008RQMDU4/ref=asc_df_B008RQMDU4/

**Comment by lincolixavier** [comment]
@SouzaHenrique Obrigado! Pode adicionar uma descrição e algum link pro livro? Compra ou download. Valeu!

**Reply by SouzaHenrique** [reply]
done !
